ws-nextjs-app/pages/[service]/send/[id]/styles.ts

```diff
@@ -2,14 +2,31 @@ import { Theme, css } from '@emotion/react';
 import pixelsToRem from '../../../../../src/app/utilities/pixelsToRem';
 
 export default {
+  background: () =>
+    css({
+      position: 'relative',
+      background: 'rgb(100, 5, 5)',
+    }),
+  gradient: () =>
+    css({
+      position: 'absolute',
+      width: '100%',
+      height: '100%',
+      backgroundImage:
+        'url("https://static.files.bbci.co.uk/core/website/assets/static/container-background-masks/billboard-ambient.d267649a6871ef14d172.jpg")',
```

Comment:
Can we achieve the gradient purely with CSS rather than using an image? I'd be a bit concerned if that image ever changed or got removed. There may be some delay whilst the image loads as well that we should avoid.

Reply:
yes we can , i actually started doing it that way initially but changed it to this based on the [PS implementation](https://static.files.bbci.co.uk/core/storybook/index.html?path=/story/containers-lists-and-collections-billboard--basic) after discussion in [this thread](https://bbc-tpg.slack.com/archives/C03S8UQJZ6U/p1714735107214129?thread_ts=1714734781.289049&cid=C03S8UQJZ6U) - apparently UX did some stress testing with purely css gradient and decided it was better to use supplied assets . I can  go back to them though and try to get some more context around what that means and explain the concerns about using the image as i agree ...